docs/index.html

```diff
@@ -8,113 +8,120 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Noto+Serif+JP&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="/css/index.css">
+  <link rel="stylesheet" href="/css/animation-book.css">
```

Comment:
CSSが相対参照だとサイト上で表示されないので、アップロード後のURLも追加しておいてほしいです！

```suggestion
  <link rel="stylesheet" href="/css/animation-book.css">
    <link rel="stylesheet" href="https://matsumana07384.github.io/wedding-menu-table//css/animation-book.css">
```

Reply:
なるほどそうなんや。ありがとう！